### ISSUE TITLE
How AD_Server Plugin works?

### ISSUE THREAD

**Issue by osvaldolivondeni** [WWBN/AVideo#8614]
**Describe the question**
I am having some difficulties with the functioning of the AD_Server plugin, which we purchased from the AVideo plugin suite.

I activate the plugin, I define in which part of the video it should be played (Show Pre-Roll ads, Show Mid-Roll ads at 25%,…), I check to show markers, I define to Show ads on : Every video), …

Next, I create an Ad in "Advertising Manager", add the advertising video…

But unfortunately when playing a video, the ad video does not play, although markers where the ad video should be played appear.

Do you have any suggestions on something I might be failing at?

Thanks alot for your help!

**Screenshots**
![image](https://github.com/WWBN/AVideo/assets/105801222/133be894-3e2a-4481-9963-40d395fbb18d)
![image](https://github.com/WWBN/AVideo/assets/105801222/8b161782-6c0e-4c35-b843-e0a4c1652137)
![image](https://github.com/WWBN/AVideo/assets/105801222/3bce8136-9965-4478-b12c-74537fd8e346)

**Comment by DanielnetoDotCom** [comment]
Some plugins auto disable ads

check and share your logs when you open a video. usually, there is a clue there.

**Comment by osvaldolivondeni** [comment]
Thanks alot for your response, @DanielnetoDotCom .

It was found that only MP4 videos work for Ads. I was uploading videos as HLS.

**Comment by DanielnetoDotCom** [comment]
Correct, unfortunately for some reason google ima does not play HLS videos on videojs
